storage_service/locations/models/package.py

```diff
@@ -317,6 +319,67 @@ def _update_quotas(self, space, location):
         location.used += self.size
         location.save()
 
+    def move_to_location(self, destination_location_uuid):
+        """
+        Move a package to another location (that has the same purpose as package's current location).
+        """
+        destination_location = Location.active.get(uuid=destination_location_uuid)
+
+        if self.current_location == destination_location:
+            return
+
+        if self.current_location.purpose != destination_location.purpose:
+            raise ValueError('Location purpose mismatch for location %s' % destination_location_uuid)
+
+        if self.status == Package.MOVING:
+            raise StorageException("Can't move package {} as it's already being moved.".format(self.uuid))
+
+        self.status = Package.MOVING
+        self.save()
+
+        try:
+            # Move to internal storage
+            ss_internal = Location.active.get(purpose=Location.STORAGE_SERVICE_INTERNAL)
+
+            source_path = os.path.join(
+                self.current_location.relative_path,
+                self.current_path)
+
+            origin_space = self.current_location.space
+            origin_space.move_to_storage_service(
+                source_path=source_path,
+                destination_path='move',
```

Comment:
also the destination path 'move' seems not being created elsewhere, so the rsync triggered `move_to_location()` produces an error if the directory `/var/archivematica/storage_service/move` was not manually created beforehand.